internal/clients/obcluster.go

```diff
@@ -89,7 +89,27 @@ func GetOBCluster(ctx context.Context, namespace, name string) (*v1alpha1.OBClus
 func DeleteOBCluster(ctx context.Context, namespace, name string) error {
 	return ClusterClient.Delete(ctx, namespace, name, metav1.DeleteOptions{})
 }
-
+func CreateOBClusterOperation(ctx context.Context, obclusterOperation *v1alpha1.OBClusterOperation) (*v1alpha1.OBClusterOperation, error) {
```

Comment:
It seems that there is no empty line in several files containing new functions. Had better add an empty line for separating function blocks in future.